rasa_sdk/tracing/utils.py

```diff
@@ -17,10 +17,11 @@ def get_tracer_provider(endpoints_file: str) -> Optional[TracerProvider]:
 
 
 def get_tracer_and_context(
-    tracer_provider: Optional[TracerProvider], request: Request
+    tracer_provider: Optional[TracerProvider], request: Union[Request]
```

Comment:
This had the WebhookRequest before I think in the typing hint, could you please re-add it?

Reply:
But that serves a different purpose, we need this for passing the rasa-private context to the sdk 🤔

Reply:
The tracer in the webhook instruments the webhook separately.